src/libs/ReportUtils.js

```diff
@@ -1089,32 +1089,46 @@ function getPersonalDetailsForAccountID(accountID) {
     );
 }
 
+function hasOnlyAvatarField(obj) {
+    const keys = _.keys(obj);
+    return keys.length === 1 && keys[0] === 'avatar';
+}
+
 /**
  * Get the displayName for a single report participant.
  *
  * @param {Number} accountID
  * @param {Boolean} [shouldUseShortForm]
+ * @param {Boolean} shouldNotFallbackToHidden
  * @returns {String}
  */
-function getDisplayNameForParticipant(accountID, shouldUseShortForm = false) {
+function getDisplayNameForParticipant(accountID, shouldUseShortForm = false, shouldNotFallbackToHidden = false) {
     if (!accountID) {
         return '';
     }
     const personalDetails = getPersonalDetailsForAccountID(accountID);
+    // check if it's invite account
+    if (hasOnlyAvatarField(personalDetails)) {
+        return '';
```

Comment:
Sure! Say a user is entering an email address into the search field, and that address has not yet been registered as Expensify user, it will, in that scenario have no other details than automatically created avatar. This is how I detect a "new invite account", because the can't fallback to "Hidden" `displayName` value.

Reply:
Thanks for the explanation @lukemorawski. Let's add a comment as a JSDoc for the method.